modules/ROOT/pages/6.7-release-notes.adoc

```diff
@@ -317,12 +317,71 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
+
+As a consequence, the Accordion element was rendered unusable: the `<details>` element requires a `<summary>` element to act as the title or heading of a specific Accordion component.
+
+In {productname} 6.7, when the user drags a `<summary>` element’s  contents out of an Accordion component, the editor puts an empty `<summary>` element back in place after the deletion consequent to the dragging event. This restores the Accordion component to its required form.
 
 === It was possible to break `<summary>` elements if content containing block elements was dragged-and-dropped inside them
 // TINY-9960
+In previous versions of the Accordion Plugin, dragging and dropping block elements within the `<summary>` element of an accordion component could unintentionally cause the `<summary>` element to break.
+
+As a consequence, the editor would duplicate the `<summary>` element.
+
+Example initial content:
+[source, html]
+——
+<details class="mce-accordion" open="open">
+<summary>Accordion summary...</summary>
+<p>Accordion body...</p>
+</details>
+<h1>block_element</h1>
+——
+
+Example of duplicated content after the drag and drop.
+[source, html]
+——
+<details class="mce-accordion" open="open">
+<summary>Accordion</summary>
+<h1>block_element</h1>
+<summary class="mce-accordion-summary">&nbsp;summary...</summary> // duplicated summary class
+<p>Accordion body...</p>
+</details>
+——
+
+{productname} 6.7, introduced a fix that now unwraps block elements after they are dragged from a block element.
+
+In {productname} 6.7, dragging and dropping content from within a block element can no longer produce duplicate summary elements by dragging in blocks into the summary element.
 
 === Contents were not removed from the drag start source if dragging and dropping internally into a transparent block element
 // TINY-9960
+Previously, when dragging and dropping a context selection from a transparent element into another transparent block element, the drag start source content would not properly be deleted after the drop event.
+
+Example: of the drag event before the drop
+[source,html]
+----
+<a href="../../undefined/">
+  <p>Transparent element</p>
+</a>
+<a href="../../undefined/">
+  <p>Transparent element</p>
+</a>
+----
+
+Example: of the drop source content remaining after the drop event
+[source,html]
+----
+<a href="../../undefined/">
+  <p>Transparent element</p> // original source not removed.
+</a>
+<a href="../../undefined/">
+  <p>Transparent element</p>
+  <p>Transparent element</p> // after the drop event.
+</a>
+----
+
+{productname} 6.7, introduced a fix that overrides this unintended behavior. Now, when dragging a content selection from one transparent element into another, removes the original source element as expected.
```

Comment:
```suggestion
Previously, when dragging and dropping a context selection from a transparent element into another transparent block element, the source content (ie the content selected for dragging) was not properly deleted after the drop event.

For example, if the initial selection for dragging was as follows:

[source,html]
----
<a href="../../undefined/">
  <p>Selected content in transparent element to be dragged.</p>
</a>
<a href="../../undefined/">
  <p>Content in another transparent element</p>
</a>
----

After the drop event, the editor state was thus:

[source,html]
----
<a href="../../undefined/">
  <p>Selected content in transparent element to be dragged.</p>
  <!-- Note the dragged content has remained in place. -->
</a>
<a href="../../undefined/">
  <p>Content in another transparent element</p>
  <p>Selected content in transparent element to be dragged.</p>
  <!-- Causing, in effect, the dragged content to be duplicated rather than moved. -->
  <!-- Also note, these comments were manually added to this example. -->
</a>
----

Note the duplicated material in the post-drag editor state example.

{productname} 6.7 includes a fix that over-rides this unintended behavior. Now, when dragging a content selection from one transparent element into another, the drag’s source selection is moved rather than duplicated, as expected.
```
Suggested edits.